tests/Controller/ConfigControllerTest.php

```diff
@@ -1223,27 +1224,69 @@ public function testUserDisable2faEmail()
 
     public function testUserEnable2faGoogle()
     {
+        $googleAuthenticatorMock = $this->createMock(GoogleAuthenticatorInterface::class);
+        $googleAuthenticatorMock
+            ->method('generateSecret')
+            ->willReturn('DUMMYSECRET');
+        $googleAuthenticatorMock
+            ->method('checkCode')
+            ->willReturnCallback(function ($user, $code) {
+                return $code === '123456';
+            });
+        
         $this->logInAs('admin');
         $client = $this->getTestClient();
+        $client->disableReboot(); // Disable reboot to keep the mock in the container
 
-        $crawler = $client->request('GET', '/config');
+        // fixme Not sure of the best way to set the mock in the container here
+        // name::class notation does not work in this context
+        //self::getContainer()->set(GoogleAuthenticatorInterface::class, $googleAuthenticatorMock);
+        self::getContainer()->set('scheb_two_factor.security.google_authenticator', $googleAuthenticatorMock);
+        //static::$kernel->getContainer()->set('scheb_two_factor.security.google_authenticator', $googleAuthenticatorMock);
```

Comment:
I have a doubt here as the `GoogleAuthenticatorInterface::class` notation does not seem to work here

Reply:
@nicosomb @yguedidi @j0k3r request for comment here